### PR TITLE
Added and updated planes and pictures.

### PR DESCRIPTION
**Pull request by pmd5700**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

File: plane_images.csv [hunk 1]
```diff
@@ -3618,7 +3618,7 @@ AE5968,https://cdn.jetphotos.com/full/6/81792_1626147109.jpg,https://cdn.jetphot
 AE5967,https://live.staticflickr.com/65535/51916058873_894d9700b7_k.jpg,https://live.staticflickr.com/65535/51650275055_b1755b575f_b.jpg,https://live.staticflickr.com/65535/51660071042_75cf6aa78c_k.jpg,
 AE5964,https://cdn.jetphotos.com/full/5/84380_1632271304.jpg,https://cdn.jetphotos.com/full/5/43756_1630638701.jpg,,
 AE5962,https://cdn.jetphotos.com/full/6/69830_1586751316.jpg,,,
-AE54D4,,,,
+AE54D4,https://cdn.jetphotos.com/full/5/22641_1433260001.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:3621]:
```suggestion
AE54D4,https://cdn.jetphotos.com/full/5/40426_1544486419.jpg,https://cdn.jetphotos.com/full/5/22641_1433260001.jpg,,
```

File: plane_images.csv [hunk 2]
```diff
@@ -8499,8 +8499,8 @@ A11E2D,,,,
 4AAA12,https://cdn.jetphotos.com/full/6/33635_1635281627.jpg,https://cdn.jetphotos.com/full/5/15721_1626000136.jpg,,
 4AAA0F,https://cdn.jetphotos.com/full/5/21514_1629053239.jpg,https://cdn.jetphotos.com/full/6/51671_1606065961.jpg,,
 4AAA0E,,,,
-A7F075,,,,
-A4D98E,,,,
+A7F075,https://live.staticflickr.com/65535/50226711088_2ad794d60c_k.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:8502]:
```suggestion
A7F075,https://live.staticflickr.com/65535/50226711088_2ad794d60c_k.jpg,https://www.airport-data.com/images/aircraft/000/200/200568.jpg,https://patch.com/img/cdn/users/428336/2011/12/raw/e22c53126dde2a2a8f5526173ff2518e.jpg,
```

File: plane_images.csv [hunk 3]
```diff
@@ -8499,8 +8499,8 @@ A11E2D,,,,
...
+A4D98E,https://photos-e1.flightcdn.com/photos/retriever/4cd6d5747b07e7e134edb3a74ec02333be9c4df3,,,
```
Comment (Phaeton) [inline plane_images.csv:8503]:
```suggestion
A4D98E,https://photos-e1.flightcdn.com/photos/retriever/4cd6d5747b07e7e134edb3a74ec02333be9c4df3,https://i.pinimg.com/originals/22/10/d0/2210d05db1a6615504822185141576f0.jpg,https://www.theledger.com/gcdn/authoring/2017/09/08/NLED/ghows-LK-571315db-d0bc-04fa-e053-0100007f7ff1-8acb3c88.jpeg,
```

File: plane_images.csv [hunk 4]
```diff
@@ -9386,12 +9386,12 @@ AE68BB,https://live.staticflickr.com/1537/23951558104_c9b20fdc41_k.jpg,https://c
 AE68BA,https://cdn.jetphotos.com/full/5/63427_1674139857.jpg,https://cdn.jetphotos.com/full/5/16999_1673272523.jpg,https://cdn.jetphotos.com/full/6/65989_1638808438.jpg,
 AE68B9,https://cdn.jetphotos.com/full/6/88984_1665723084.jpg,https://cdn.jetphotos.com/full/6/36469_1663227926.jpg,https://cdn.jetphotos.com/full/6/83911_1657719694.jpg,https://cdn.jetphotos.com/full/6/78233_1649470096.jpg
 AE68B8,https://cdn.jetphotos.com/full/6/38224_1659441712.jpg,https://cdn.jetphotos.com/full/5/65231_1626807138.jpg,https://cdn.jetphotos.com/full/5/14056_1417960541.jpg,https://cdn.jetphotos.com/full/4/14953_1383241418.jpg
-AE68B7,,,,
+AE68B7,https://cdn.jetphotos.com/full/5/45720_1529583255.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:9389]:
```suggestion
AE68B7,https://cdn.jetphotos.com/full/5/45720_1529583255.jpg,https://cdn.jetphotos.com/full/5/64316_1465055234.jpg,https://cdn.jetphotos.com/full/5/33308_1529676594.jpg,https://cdn.jetphotos.com/full/6/99114_1614046694.jpg
```

File: plane_images.csv [hunk 5]
```diff
@@ -9386,12 +9386,12 @@ AE68BB,https://live.staticflickr.com/1537/23951558104_c9b20fdc41_k.jpg,https://c
...
 AE68B6,,,,
 AE68B5,https://cdn.jetphotos.com/full/3/75169_1351003679.jpg,https://cdn.jetphotos.com/full/6/56927_1515706533.jpg,https://cdn.jetphotos.com/full/6/95338_1438607330.jpg,https://cdn.jetphotos.com/full/6/32738_1650433323.jpg
 AE68B4,https://cdn.jetphotos.com/full/5/90713_1623259955.jpg,https://cdn.jetphotos.com/full/5/38655_1624133752.jpg,https://cdn.jetphotos.com/full/5/84645_1591072518.jpg,https://cdn.jetphotos.com/full/6/95396_1573238711.jpg
 AE68B3,https://cdn.jetphotos.com/full/6/56895_1652145427.jpg,https://cdn.jetphotos.com/full/5/59961_1646623803.jpg,https://cdn.jetphotos.com/full/5/60417_1640776440.jpg,https://cdn.jetphotos.com/full/5/79406_1638768967.jpg
-AE68B2,,,,
+AE68B2,https://cdn.jetphotos.com/full/6/92910_1401189475.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:9394]:
```suggestion
AE68B2,https://cdn.jetphotos.com/full/6/92910_1401189475.jpg,https://cdn.jetphotos.com/full/5/700256_1690062958.jpg,https://cdn.jetphotos.com/full/5/48746_1626984920.jpg,https://cdn.jetphotos.com/full/6/96579_1509482351.jpg
```

File: plane_images.csv [hunk 6]
```diff
@@ -11342,7 +11342,7 @@ AE152F,https://cdn.jetphotos.com/full/5/18234_1658053279.jpg,https://cdn.jetphot
 AE1447,,,,
 AE1446,,,,
 AE1445,,,,
-AE1444,,,,
+AE1444,https://cdn.jetphotos.com/full/6/79168_1631851821.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:11345]:
```suggestion
AE1444,https://cdn.jetphotos.com/full/6/79168_1631851821.jpg,https://cdn.jetphotos.com/full/6/92728_1653231621.jpg,https://cdn.jetphotos.com/full/5/98913_1526911193.jpg,
```

File: plane_images.csv [hunk 7]
```diff
@@ -11889,8 +11889,8 @@ AE54D2,https://cdn.jetphotos.com/full/5/90835_1644424763.jpg,https://cdn.jetphot
 AE54D1,,,,
 AE54CA,,,,
 AE54C6,https://cdn.jetphotos.com/full/5/18666_1530700933.jpg,https://cdn.jetphotos.com/full/5/26089_1528233017.jpg,https://cdn.jetphotos.com/full/6/68409_1507479920.jpg,
-AE4E18,,,,
-AE4E17,,,,
+AE4E18,https://cdn.jetphotos.com/full/6/15819_1581586079.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:11892]:
```suggestion
AE4E18,https://cdn.jetphotos.com/full/6/15819_1581586079.jpg,https://cdn.jetphotos.com/full/5/49285_1649926229.jpg,https://live.staticflickr.com/65535/51729523968_b1c6f0df43_b.jpg,https://live.staticflickr.com/4409/35908362634_062d21163b_b.jpg
```

File: plane_images.csv [hunk 8]
```diff
@@ -11889,8 +11889,8 @@ AE54D2,https://cdn.jetphotos.com/full/5/90835_1644424763.jpg,https://cdn.jetphot
...
+AE4E17,https://cdn.jetphotos.com/full/5/35659_1632743733.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:11893]:
```suggestion
AE4E17,https://cdn.jetphotos.com/full/5/35659_1632743733.jpg,https://cdn.jetphotos.com/full/6/44972_1554090003.jpg,https://cdn.jetphotos.com/full/5/99250_1561606883.jpg,https://cdn.jetphotos.com/full/5/15771_1581018202.jpg
```

File: plane_images.csv [hunk 9]
```diff
@@ -12298,7 +12298,7 @@ AE11CD,https://cdn.jetphotos.com/full/6/28564_1508086385.jpg,https://cdn.jetphot
 AE11CC,,,,
 AE11CB,,,,
 AE11CA,,,,
-AE11C9,,,,
+AE11C9,https://cdn.jetphotos.com/full/6/44122_1598315009.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:12301]:
```suggestion
AE11C9,https://cdn.jetphotos.com/full/6/44122_1598315009.jpg,https://cdn.jetphotos.com/full/6/59779_1421883370.jpg,https://live.staticflickr.com/8739/16713124707_b937b6399d_b.jpg,
```

File: plane_images.csv [hunk 10]
```diff
@@ -12442,7 +12442,7 @@ AE0889,https://cdn.jetphotos.com/full/6/62982_1606072980.jpg,https://cdn.jetphot
 AE0888,https://cdn.jetphotos.com/full/6/78358_1624502544.jpg,https://cdn.jetphotos.com/full/1/26472_1152800249.jpg,,
 AE0887,,,,
 AE0885,https://live.staticflickr.com/65535/52195852941_90a61ab3c1_z.jpg,,,
-AE0884,,,,
+AE0884,https://imgproc.airliners.net/photos/airliners/8/3/3/2444338.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:12445]:
```suggestion
AE0884,https://imgproc.airliners.net/photos/airliners/8/3/3/2444338.jpg,https://www.airfighters.com/photo_9999_141429.jpg,https://www.aeroresource.co.uk/wp-content/uploads/2017/02/Pensacola-USAF-no2.jpg,
```

**Review by Phaeton — CHANGES_REQUESTED** [review]
Added additional photos for the existing planes. Did not review the new adds.

**Comment by dziban303** [comment]
The Air Force does not fly E-6s, that's the Navy.

**Comment by pmd5700** [comment]
> The Air Force does not fly E-6s, that's the Navy.

Woops....

**Review by pmd5700 — COMMENTED** [review]
all added manually

**Review by Phaeton — APPROVED** [review]

**Review by dziban303 — APPROVED** [review]

**Pull request merged**